spaces/S000174/README.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: S000174
+slug: pol-space
+name: Pol's space
```

Comment:
Many spaces were constructed by Pol.  I think it would be helpful to be a little more specific in the title, in case we want to add other examples in the future. Maybe something like "Pol's $T_6$ non-paracompact space".